lib/rack/prx_auth/auth_validator.rb

```diff
@@ -35,11 +35,18 @@ def decode_token
       end
 
       def expired?
-        now = Time.now.to_i - 30 # 30 second clock jitter allowance
-        if claims['iat'] <= claims['exp']
-          now > claims['exp']
+        (time_to_live + 30) <= 0 # 30 second clock jitter allowance
+      end
+
+      def time_to_live
+        now = Time.now.to_i
+        if claims['exp'].nil?
+          0
+        elsif claims['iat'].nil? || claims['iat'] <= claims['exp']
```

Comment:
I may be having trouble with order of ops here, so this is "if issued at nil or has a value less than or equal expired"